### PR TITLE
Change pcp script

### PR DESCRIPTION
**Pull request by zgalor**
Change pcp-ping script to use ops-metric-sender instead of ops-zagg-sender
Depends on #1650 and #1747

### CODE REVIEW

**Comment by zgalor** [comment]
rebased after https://github.com/openshift/openshift-tools/pull/1792 was merged.
@joelddiaz @twiest please review

**Review by twiest — APPROVED** [review]
@zgalor This looks great!

This is ready to merge right after our holiday freeze is lifted.

**Pull request merged**
